content/_guides/bots.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Network bots
+category: chanop
+---
+
+We run various IRC bots, some of them have a public facing usage.
+Any requests or questions about our utility bots can be asked in `#libera-bots`.
+
+## ozone
+
+ozone is an instance of [Sigyn](https://github.com/Libera-Chat/Sigyn).
```

Comment:
I think we should start this off explaining that ozone is a network-run anti-spam bot. Many people won't know what Sigyn is.